content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Mobile Session Replay Privacy Options
+kind: documentation
+description: Describes how to configure privacy options for Mobile Session Replay
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay/mobile'
+      tag: Documentation
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Overview
+
+Session Replay provides privacy controls to ensure organizations of any scale do not expose sensitive or personal data. Data is stored on Datadog-managed cloud instances and encrypted at rest.
+
+Default privacy options for Session Replay are designed to protect end user privacy and prevent sensitive organizational information from being collected.
+
+By enabling Mobile Session Replay, you can automatically mask sensitive elements from being recorded through the RUM Mobile SDK. When data is masked, that data is not collected in its original form by Datadog's SDKs and thus is not sent to the backend.
+
+## Configuration
```

Comment:
@mariusc83 placeholder -- can you drop in the code snippet here for what a user would need to do to override the default masking setttings?

Reply:
tabs to do it for both iOS and android if there are differences